system_strings/sysmes_text.en

```diff
@@ -7264,12 +7268,12 @@ Of course it is.
 // @flowchart_descr
 The Luminary advances toward destruction.
  The angel's wheel remains.
- The manifestation of the cosmic―――
+ The manifestation of the cosmic――
 }
 [sha:b6452bee286dcdb6c6e45350f29822b66c4911f1]{
 -- 吸血姫との和解と別離。^けれど、星の光で焼けた体は、^もう保たなかった。
 // @flowchart_descr
-I reconcile with the vampiress, and say my farewells.
+I reconcile with the vampiress, and say our farewells.
```

Comment:
He can't say both of their farewells, he only says his

Reply:
I was supposed a we after and would that be better or the original is fine?
>I reconcile with the vampiress, and we say our farewells.

Reply:
Sure